MarkMode: Alone
ThrVal: 210
MarkHeight: 1440
IDNum: 3
RefID: 114
ThrAlg: Auto
MarkType: Home
MarkDim: 1
MapMode: Stop
